port/unix/omrsysinfo.c

```diff
@@ -1602,6 +1626,48 @@ omrsysinfo_get_memory_info(struct OMRPortLibrary *portLibrary, struct J9MemoryIn
 	return rc;
 }
 
+int32_t
+omrsysinfo_get_cpu_info(struct OMRPortLibrary *portLibrary, struct J9CpuInfo *cpuInfo, ...)
+{
+	int32_t rc = -2;
+	
+	if (NULL == cpuInfo) {
+		return OMRPORT_ERROR_SYSINFO_NULL_OBJECT_RECEIVED;
+	}
+	/* Initialize to defaults. */
+	cpuInfo->cpuQuota = OMRPORT_CPUINFO_NOT_AVAILABLE;
+	cpuInfo->cpuPeriod = OMRPORT_CPUINFO_NOT_AVAILABLE;
+	cpuInfo->numCpusQuota = (int32_t) OMRPORT_CPUINFO_NOT_AVAILABLE;
+	cpuInfo->numCpusBasedOnCpuSet = (int32_t) OMRPORT_CPUINFO_NOT_AVAILABLE;
+
+#if defined(LINUX)
+	int64_t cpuQuota = 0;
```

Comment:
This is not legal C code: it will not compile (with MSVC); variables must be declared at the beginning of a block.